streams/src/test/java/org/apache/kafka/streams/integration/SuppressionDurabilityIntegrationTest.java

```diff
@@ -179,17 +190,17 @@ public void shouldRecoverBufferAfterShutdown() {
             );
             verifyOutput(
                 outputRaw,
-                asList(
-                    new KeyValueTimestamp<>("v4", 1L, scaledTime(4L)),
-                    new KeyValueTimestamp<>("v5", 1L, scaledTime(5L))
-                )
+                new HashSet<>(asList(
+                    new KeyValueTimestamp<>("k4", 1L, scaledTime(4L)),
+                    new KeyValueTimestamp<>("k5", 1L, scaledTime(5L))
```

Comment:
Instead of grouping on the values anymore, now I'm just counting events. I just didn't realize that simpler aggregation was available when I wrote the tests the first time.